tests.webpack.js

```diff
@@ -1,2 +1,2 @@
-const context = require.context('./src', true, /\.test\.js$/)
+const context = require.context('./test', true, /\.test\.js$/)
```

Comment:
Why do you want to keep the file in `src/index.test.js`? For me this indicates that it is an `unit test`  _only_ for the `index file`, but it is an `integration test` for the whole project. For me that's a bit confusing. What if you were to add unit tests in the future?